libs/dotcms-webcomponents/src/components/contenttypes-fields/dot-key-value/components/key-value-table/key-value-table.scss

```diff
@@ -1,21 +1,22 @@
 key-value-table {
-
     table {
         width: 100%;
     }
 
     .key-value-table-wc__key {
-        padding-left: 5%;
         width: 45%;
     }
 
     .key-value-table-wc__value {
-        padding-left: 9%;
         width: 45%;
     }
 
     .key-value-table-wc__action {
         text-align: right;
         width: 10%;
     }
+
```

Comment:
Flex!

Reply:
does not apply, this css class is asigned to a td tag from a table